config/site_settings.yml

```
@@ -3973,3 +3973,6 @@ experimental:
     client: true
     default: false
     hidden: true
+  experimental_content_localization:
```

This is already under the `experimental` site setting group so I'm not sure if we need to repeat `experimental` in the site setting's name.

```suggestion
  content_localization:
```